javascript/javascript1/week2/studman.js

```diff
@@ -0,0 +1,23 @@
+const class07Students = [];
+const maxStudents = 6;
+
+function addStudentToClass(studentName) {
+    if (!studentName) {
+        return "Student name cannot be empty.";
+    }
+
+    if (class07Students.includes(studentName)) {
+        return "Student" + studentName +  "is already in the class.";
+    }
+
+    if (class07Students.length >= maxStudents && studentName === "Queen") {
```

Comment:
Did you remember to test all the cases highlighted in the homework? 🧪 I think our Queens will be very disappointed if they try to sign up for a full class. 🥺 On the contrary, everyone else will be delighted!  Can you spot the bug? 🐛

Reply:
oh no let me fix it and save the queen😓